pom.xml

```diff
@@ -89,6 +115,12 @@
             <artifactId>connect-json</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.parquet</groupId>
```

Comment:
we don't need this block, we get it from `storage-common`. Also we shouldn't mark it as `provided` because the standalone archive of the connector (the zip that is constructed by the `kafka-connect-maven-plugin`) will not have all it needs.